src/package.json

```
@@ -0,0 +1,22 @@
+{
+  "name": "demo.identityserver",
```

```suggestion
  "name": "demo.duendesoftware.com",
```

To not link to non-Duende by accident